tools/InterProScan.md

```diff
@@ -0,0 +1,139 @@
+# InterProScan
+written by: [Katarina Vucic](https://github.com/katarinaavucic)
+
+[15 minutes] This tutorial introduces InterProScan, a tool designed to identify protein domains within a given protein sequence. InterProScan works by matching input sequences against InterPro protein signature databases, providing detailed reports on predicted functional domains, including their superfamily information, where they map to amino acid sequences, the significant (E-values), and related pathways. This tutorial specifically focuses on using InterProScan for Virus Discovery, using the example of an uncharacterized virus with a potential Narnavirus classification.
+
+**Tutorial Objective**: By the end of this tutorial, you will be able to use InterProScan to identify potential protein domains in a given protein sequence.
+
+## Input / Prerequisites
+- [Tool Weblink](https://www.ebi.ac.uk/interpro/result/InterProScan/)
+- [Tutorial Data](img/template/ao.fa)
+- In order to use InterProScan to analyse your virus, you need a protein sequence of the region you would like to analyse. For this tutorial, we will be using the protein sequence containing the RdRp (RNA-dependent RNA polymerase). The protein sequence used in this tutorial is provided in the dropbox, but if you would like to try with your own virus the steps are as follows:
```

Comment:
in the dropbox?

Reply:
Updated this so now the path leads to the file